metadata.json

```diff
@@ -58,15 +58,5 @@
         "25"
       ]
     }
-  ],
```

Comment:
Only the `pe` requirement is deprecated, the `puppet` requirement is still valid.
The block to remove is:
```
-    {		
 -      "name": "pe",		
 -      "version_requirement": ">= 3.0.0"		
 -    },
```
Sorry for not being clear enough!